soc/arm/nxp_imx/rt6xx/Kconfig.series

```diff
@@ -10,5 +10,6 @@ config SOC_SERIES_IMX_RT6XX
 	select CPU_CORTEX_M_HAS_DWT
 	select SOC_FAMILY_IMX
 	select CLOCK_CONTROL
+	select CODE_DATA_RELOCATION_SRAM
```

Comment:
Since we only relocate files to SRAM when `CONFIG_CODE_FLEXSPI=y`, this needs to be set to `select CODE_DATA_RELOCATION_SRAM if CODE_FLEXSPI`

Reply:
Thank you, I have added this change.